src/linter.ts

```diff
@@ -25,10 +24,11 @@ export const lint = (fileName: string, configuration = defaultConfiguration) =>
       return;
     }
 
-    doNotUseBreakingChanges(changeSet, reporter);
+    rules.forEach(checkRule => {
+      checkRule(changeSet, reporter)
```

Comment:
not very consistent on the use of semicolons ;)

Reply:
Hahaha si seulement le propriétaire de ce repo était un génie du prettier !